docs/cheat-sheets/java-xxe.mdx

```diff
@@ -0,0 +1,341 @@
+| setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true)                            | ✅⚠️       | ✅⚠️  | ✅⚠️               | ✅⚠️  | ✅      | ✅       | ✅        | ✅⚠️  | ✅      | ✅       |
+| setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "")                                  | n/a      | n/a | n/a              | n/a | n/a    | n/a     | n/a      | n/a | n/a    | n/a     |
+| setAttribute(XMLConstants.ACCESS_EXTERNAL_SCHEMA, "")                               | n/a      | n/a | n/a              | n/a | n/a    | n/a     | n/a      | n/a | n/a    | n/a     |
+| setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "");                          | n/a      | n/a | n/a              | n/a | n/a    | n/a     | n/a      | n/a | n/a    | n/a     |
+| setFeature("http://apache.org/xml/features/disallow-doctype-decl", true)            | ✅⚠️       | ✅⚠️  | ✅⚠️               | ✅⚠️  | ✅      | ✅       | ✅        | ✅⚠️  | ✅      | ✅       |
+| setFeature("http://xml.org/sax/features/external-general-entities", false)          | ✅⚠️       | ✅   | ❌                | ✅   | ✅      | ✅       | ✅        | ✅   | ✅      | ✅       |
+| setFeature("http://xml.org/sax/features/external-parameter-entities", false)        | ✅⚠️       | ❌   | ✅                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
+| setValidating(false)                                                                | ✅⚠️       | ❌   | ❌                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
+| setExpandEntityReferences(false)                                                    | n/a      | n/a | n/a              | n/a | n/a    | n/a     | n/a      | n/a | n/a    | n/a     |
+| setXIncludeAware(false)                                                             | ✅⚠️       | ❌   | ❌                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
+| setFeature("http://apache.org/xml/features/xinclude", false)                        | ✅⚠️       | ❌   | ❌                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
+| setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false) | ✅⚠️       | ❌   | ❌                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
+| setFeature("http://apache.org/xml/features/nonvalidating/load-dtd-grammar", false)  | n/a      | n/a | n/a              | n/a | n/a    | n/a     | n/a      | n/a | n/a    | n/a     |
+| setFeature("http://apache.org/xml/features/namespaces", false)                      | n/a      | n/a | n/a              | n/a | n/a    | n/a     | n/a      | n/a | n/a    | n/a     |
+| setNamespaceAware(true)                                                             | n/a      | n/a | n/a              | n/a | n/a    | n/a     | n/a      | n/a | n/a    | n/a     |
+
+### 3.D SAXParser
```

Comment:
I am really not sure about the structure here... if we need these tables only to accompany the blog post, this can be done in r2c blog also. I am not sure about numbering here...